apps/jamtools/server/tsconfig.json

```diff
@@ -0,0 +1,115 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+    "jsx": "react",
+    "paths": {
+      "@/*": ["./*"]
+    },
+    "baseUrl": "./src",
```

Comment:
This branch doesn't have the consolidated stuff from the module system branch. We should do that instead of duplicating config here